t/app/controller/admin.t

```diff
@@ -1029,13 +1020,12 @@ subtest 'hiding comment marked as fixed reopens report' => sub {
     $report->state('fixed - user');
     $report->update;
 
-
     my $fields = {
             text => 'this is a changed update',
             state => 'hidden',
             name => 'A User',
             anonymous => 0,
-            email => 'test2@example.com',
+            username => 'test2@example.com',
     };
 
     $mech->submit_form_ok( { with_fields => $fields } );
```

Comment:
Tests using phone numbers would be nice.

Reply:
They would wouldn't they. I think add user form isn't tested at all, so can do two birds with one stone hopefully.